src/main/java/net/dries007/tfc/objects/inventory/slot/SlotQuernOutput.java

```diff
@@ -0,0 +1,85 @@
+/*
```

Comment:
Same as above, all of this should be done with `SlotCallback`. As for the xp, I'm thinking we should just have the quern not produce xp for now. Reason being: no other devices produce xp at the moment, and I'd rather keep things consistent for the time being.